easybuild/easyconfigs/h/HarfBuzz/HarfBuzz-4.2.1-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
```

Comment:
Same here with the author info, though fine to note that it was for previous versions.

Reply:
the credit was present for one version, and that isn't the most recent?

Reply:
Yes, it isn't always super-clear in the `review` output, and as above it looks like it was already removed.

I prefer to check this, as I know for some people it's a condition of their contributions that they add the authorship.